### PR TITLE
Update Imperative and integration test snapshots

### PR DESCRIPTION
**Pull request by t1m0thyj**
Signed-off-by: Timothy Johnson <timothy.johnson@broadcom.com>

### CODE REVIEW

**Review by traeok — APPROVED** [review]

**Review by traeok — CHANGES_REQUESTED** [review]
Looks like a couple snapshots are still failing, will approve once fixed

File: package.json [hunk 1]
```diff
@@ -51,7 +51,7 @@
     "@typescript-eslint/parser": "^5.33.0",
     "@zowe/cli": "^7.9.7",
     "@zowe/cli-test-utils": "^7.9.7",
-    "@zowe/imperative": "^5.8.0",
+    "@zowe/imperative": "^5.8.1",
```
Comment (gejohnston) [inline package.json:54]: Isn't imperative now up to 5.8.2?

Comment (t1m0thyj) [reply package.json:54]: Yes, the Imperative version is updated to 5.8.2 in npm-shrinkwrap, but the minimum version required to update the snapshots was 5.8.1.

Comment (gejohnston) [reply package.json:54]: ok. Thanks for the clarification.

**Review by gejohnston — APPROVED** [review]

**Review by traeok — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]

**Pull request merged**